sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/RealTimeNotificationCallback.java

```diff
@@ -0,0 +1,9 @@
+package com.azure.android.communication.chat.signaling;
+
+import org.json.JSONObject;
+
+public interface RealTimeNotificationCallback {
+
+    public void onChatEvent(JSONObject chatEvent);
```

Comment:
If we are not expecting to add more methods to this interface, then we can mark it as `FunctionalInterface` so that end user can use lambda.